docs/outlook/outlook-on-send-addins.md

```diff
@@ -16,6 +16,14 @@ The on-send feature is triggered by the `ItemSend` event type and is UI-less.
 
 For information about limitations related to the on-send feature, see [Limitations](#limitations) later in this article.
```

Comment:
In the AppSource bullet of the Limitations section, I think it would be helpful to include a link to the comparison table or to the store options article. Maybe append something like: "However, Smart Alerts is a newer feature which can provide AppSource support for similar functionality. To learn more, see *{some link}*."

Reply:
Thanks for the suggestion. Links have been added. I linked to the overall Smart Alerts walkthrough, instead of the comparison table, so that the user gets an introduction to the feature before diving into specific details.